kube/services/netpolicy/gen3/services/redis_netpolicy.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
```

Comment:
I have not written a network policy in k8s before so I am happy to get any guidance on this.